src/app/shell/admin-tools/platform/create-direction/directions/directions.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="header-wrapper" fxLayout="row wrap" fxLayoutAlign="space-between center">
+  <div class="search shadow">
+    <mat-form-field floatLabel=never appearance="none" fxLayout='row' fxLayoutAlign='space-between center'>
+      <input type="text" maxlength="200" [formControl]="searchValue" matInput placeholder="Введіть назву напрямку"
+        (keyup.enter)="onSearch()" >
+      <mat-error></mat-error>
+      <mat-icon class="icon" style="transform: rotate(90deg)" (click)="onSearch()">search</mat-icon>
+    </mat-form-field>
+  </div>
+  <a routerLink="./create"><button class="btn" mat-raised-button>Додати напрямок</button></a>
+</div>
+<div class="card-wrapper" *ngIf="(filteredDirections$ | async); else isEmptyList">
+  <app-category-card class="card" *ngFor="let direction of (filteredDirections$ | async)?.entities " [direction]="direction"
+    [isEditMode]="true" (deleteDirection)="onDelete($event)">
+  </app-category-card>
+</div>
+  <app-paginator  [totalEntitiesDir]="(filteredDirections$ | async).totalAmount" [currentPage]="currentPage"
```

Comment:
instead totalEntitiesDir use the old vvalue
I don't see why do we need a new input